script.js

```diff
@@ -0,0 +1,97 @@
+const API_KEY = 'b6ad49d4';
+const RESULTS_PER_PAGE = 10;
+
```

Comment:
If there is no image available - use a fallback image or a text with "No image provided"

<img width="1465" alt="Screenshot 2022-11-18 at 11 46 27" src="https://user-images.githubusercontent.com/34886027/202672091-a194e8a3-c380-44d4-a673-c0ab5514e967.png">

Reply:
And I would like to see more details about the movie. Not only image, year and name :)